app/src/main/java/com/udacity/shoestore/ShoeDetailFragment.kt

```diff
@@ -44,8 +47,6 @@ class ShoeDetailFragment : Fragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
         save_fab.setOnClickListener {
-            val action = ShoeDetailFragmentDirections.actionShoeDetailFragmentToShoeListFragment()
-            findNavController().navigate(action)
```

Comment:
I moved the navigation until _after_ we'd updated the `viewModel` with the new `Shoe`, just in case navigating interrupted the update. It also reads more logically ("first, create the new shoe and save it; then navigate back to the list screen").